packages/ember-htmlbars/lib/utils/string.js

```diff
@@ -38,8 +39,39 @@ if (ENV.EXTEND_PROTOTYPES.String) {
   };
 }
 
+/**
+  Detects if a string was decorated using `Ember.String.htmlSafe`.
+
+  ```javascript
+  var plainString = 'plain string',
+      safeString = Ember.String.htmlSafe('<div>someValue</div>');
+
+  Ember.String.isHtmlSafe(plainString); // false
+  Ember.String.isHtmlSafe(safeString);  // true
+  ```
+
+  @method isHtmlSafe
+  @for Ember.String
+  @static
+  @return {Boolean} `true` if the string was decorated with `htmlSafe`, `false` otherwise.
+  @public
+*/
+function isHtmlSafe(str) {
+  return str && typeof str.toHTML === 'function';
+}
+
+if (isEnabled('ember-string-ishtmlsafe')) {
+  EmberStringUtils.isHtmlSafe = isHtmlSafe;
+  if (ENV.EXTEND_PROTOTYPES.String) {
+    String.prototype.isHtmlSafe = function() {
```

Comment:
As far as I can see the RFC (https://github.com/emberjs/rfcs/blob/master/text/0139-isHtmlSafe.md) does not suggest adding this.  I think we should avoid adding another thing that modifies `String.prototype`.  Can you remove this block?